book/src/configuration.md

```diff
@@ -83,7 +97,6 @@ The following elements can be configured:
 | `selections` | The number of active selections |
 | `position` | The cursor position |
 | `position-percentage` | The cursor position as a percentage of the total number of lines |
-| `separator` | The string defined in `editor.statusline.separator` (defaults to `"│"`) |
```

Comment:
It looks like this may have been deleted by accident? The separator is both a key in `editor.statusline` and an element you can use in left/center/right

Reply:
Just fixed that, thanks for the notice!